### ISSUE TITLE
Undocumented python-paramiko dependency?

### ISSUE THREAD

**Issue by dmesser** [pcuzner/ceph-ansible-copilot#2]
On RHEL 7 I had to install python-paramiko to get rid of this during setup.py install:

```
...utils/utils.py", line 13, in <module>
from paramiko import SSHClient, AutoAddPolicy AutoAddPolicy
ImportError: No module named paramiko
```

**Comment by pcuzner** [comment]
@dmesser Thanks!

I've updated the README in the 0.9.2 branch. More updates there to come, since 0.9.2 brings in host and cluster validation rules.

**Comment by pcuzner** [comment]
Closing - issue fixed in 0.9.2 and further clarification added in 0.9.5
